src/views/Download.vue

```diff
@@ -1,13 +1,38 @@
 <template>
-  <v-card class="downloads" flat>
-    <v-card-text>
-      Lorem ipsum dolor sit amet.
-    </v-card-text>
-  </v-card>
+  <v-container class="download">
+    <v-row>
+      <v-col>
+        <v-btn
+          :color="drawMode === 'rectangle' ? 'primary' : null"
```

Comment:
I would move this logic to a computed prop and leave as much logic like this out of our templates.